manim/cli/render/commands.py

```diff
@@ -129,7 +129,8 @@ def __repr__(self):
         req_info = {}
 
         try:
-            req_info = requests.get(manim_info_url)
+            # OpenRefactory Warning: The 'requests.get' method does not use any 'timeout' threshold which may cause program to hang indefinitely.
+            req_info = requests.get(manim_info_url, timeout=100)
```

Comment:
10 seconds is sufficiently long for a timeout.

Actually, it would be nice to implement some sort of local caching for this so that this doesn't have to run every single time.

```suggestion
            req_info = requests.get(manim_info_url, timeout=10)
```